### PR TITLE
Merge with v1.3.6

### PR DESCRIPTION
**Pull request by vtsaplin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Change Log
+
+## [v1.3.5](https://github.com/* remote origin
```
Comment (ramboz) [inline CHANGELOG.md]: Missing the 1.3.6 changelog ;) Would be nice to have it also

Comment (vtsaplin) [reply CHANGELOG.md]: They tagged first and then added an entry to a change log :-) I merged this commit as well. Not much has changed through..

**Review by ramboz — APPROVED** [review]
Looks ok at first glance. Hard to clearly judge though since I can't identify what are our changes and what are theirs

**Review by basecode — APPROVED** [review]
I compared this branch with upstream: https://github.com/phonegap/phonegap-plugin-contentsync/compare/master...adobe-rnd:merge-with-v1.3.6#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

Great job! You just earned "mastering merge-conflicts" badge! We shouldn't have touched files like `package.json` or `plugin.xml` in the first place. We're not respecting the linting rules of upstream project. Might have been IDE issue.

I did not pay too much attention to the macOS changes because there's a follow-up PR in the works. See: https://github.com/adobe-rnd/phonegap-plugin-contentsync/pull/2

**Review by tripodsan — APPROVED** [review]
I cannot really judge if the changes are correct. there is one PR pending on the upstream which we added to our code already, https://github.com/phonegap/phonegap-plugin-contentsync/pull/97, but I'm not sure if we still need it.

**Pull request merged**